src/assets/scripts/client/airport/AirportController.js

```diff
@@ -205,6 +212,7 @@ class AirportController {
      * @method getInitialArrivalRunwayName
      * @return {string}
      */
+
```

Comment:
no empty lines between dockblocks and methods. could you please remove this line?